README.md

```diff
@@ -229,3 +229,4 @@ maintenance processes.
 *   PostgreSQL in-database types: https://github.com/blm768/pg-libphonenumber
 *   Python: https://github.com/daviddrysdale/python-phonenumbers
 *   Ruby: https://github.com/mobi/telephone_number
+*   Android: https://github.com/MichaelRocks/libphonenumber-android
```

Comment:
1. I would say Android-optimized (since our port is actually also used in Android.

2. It would be nice to include the info you had in the PR description about assets as well. Something like the wording used to describe the other Javascript versions.

3. Please put this first to keep the alpha sorting

Reply:
Updated the PR with the comments. Calling it Android so it's similar to JS ports entry.